app/serializers/payload_serializer.rb

```diff
@@ -0,0 +1,10 @@
+class PayloadSerializer < ActiveModel::Serializer
+  def attributes
+    data = super
+    data[:id]          = object.id
+    data[:url]         = object.url
+    data[:referrer]    = object.referrer if object.referrer.present?
+    data[:created_at]  = object.created_at
+    data
+  end
+end
```

Comment:
Since the referrer is conditional, I choose to format the serializer by overriding the attributes method. This way I can keep it readable and display the payload in the same format the user would expect (the referrer before the created_at).